modules/administration-guide/pages/configuring-namespace-provisioning.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: CONCEPT
+:navtitle: Configuring {orch-namespace}s
+:keywords: administration guide, configuring, namespace
+:page-aliases: installation-guide:configuring-namespace-strategies, configuring-namespace-strategies
+
+[id="configuring-user-{orch-namespace}-provisioning_{context}"]
+= Configuring user {orch-namespace} provisioning
+
+For each user, {prod-short} isolates the workspaces in a {orch-namespace}. 
```

Comment:
```suggestion
For each user, {prod-short} isolates workspaces in a {orch-namespace}. 
```
No need for a definite article because you're speaking about a general function.